### DISCUSSION TITLE
Does xdrip+ sharing work during a trip to China?

### DISCUSSION THREAD

**Discussion by kapet** [NightscoutFoundation/xDrip#3372] [Q&A]
Hey,

Kid traveling to China for two weeks, we are using xdrip+ with both "xDrip+ Sync Follower" as well as nightly uploads to Tidepool to keep track of the blood sugar.

Does anyone here know if those methods will still work while in China? (True mainland China - Shanghai area.)

IIUC the xDrip+ Sync is based on Google/Firebase Cloud Messaging, which seems to only work "best effort" or "no" from what I could find out.

But I could not find any info about Tidepool.

Any information, also about other sharing methods in relation to China, would be welcome.

Thanks!
Karsten

**Comment by qq254344379** [comment]
You can log in normally，Tidepool

**Comment by qq254344379** [comment]
China can't link Google directly. If it's a non-local network, it may be possible. Tidepool has no problem linking normally
